app/tests/integration/nimbus/pages/experimenter/summary.py

```diff
@@ -155,3 +182,19 @@ def clone_save(self):
     def clone(self):
         self.clone_action.click()
         self.clone_save.click()
+
+    @property
+    def promote_to_rollout_buttons(self):
+        self.wait.until(
+            EC.presence_of_all_elements_located(self._clone_action_locator),
+            message="Summary Page: could not find promote to rollout buttons",
+        )
+        return self.find_elements(*self._promote_rollout_locator)
+
+    def promote_first_branch_to_rollout(self):
+        self.promote_to_rollout_buttons[0].click()
+        random_chars = "".join(
+            random.choices(string.ascii_uppercase + string.digits, k=6)
+        )
```

Comment:
Just gonna steal this random chars thing from legacy tests and call it good for now